Sources/StreamChat/APIClient/Endpoints/Payloads/MessagePayloads.swift

```diff
@@ -92,7 +92,7 @@ class MessagePayload<ExtraData: ExtraDataTypes>: Decodable {
         reactionScores = try container
             .decodeIfPresent([String: Int].self, forKey: .reactionScores)?
             .mapKeys { MessageReactionType(rawValue: $0) } ?? [:]
-        attachments = try container.decode([AttachmentPayload].self, forKey: .attachments)
+        attachments = try container.decode([OptionalDecodable<AttachmentPayload>].self, forKey: .attachments).compactMap(\.base)
```

Comment:
Please add a comment explaining what is happening here.

Reply:
Fixed.